__tests__/api.test.js

```diff
@@ -58,4 +59,53 @@ describe("/api/getCategories", () => {
         });
       });
   });
+
+  it("error handling, expect 404 custom error if bad path", () => {
+    return request(app)
+      .get("/api/categories/funny")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          "Path Not Found. The correct path is /api/categories"
+        );
+      });
+  });
+});
+////////////////////// API GET REVIEW OBJECT BY ID//////////////////////////////
+
+describe("/api/getReviewObjectById", () => {
+  it("return a sinlge entity in the form of an object", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toEqual(false);
+        expect(typeof body === "object").toEqual(true);
+      });
+  });
+
+  it("return correct object given correct ID", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.category).toEqual(expect.any(String));
```

Comment:
Because we know that this will specifically be review 3, and we have access to that in our test database, we can be more explicit here - we can say we literally expect an object to equal ---> and then copyin the object exactly as we expect.

Reply:
fixed :)